IS7024/Pages/HollywoodBowl.cshtml.cs

```diff
@@ -37,7 +37,7 @@ public void OnGet()
                 IList<string> validationEvents = new List<string>();
                 //if (jsonObject.IsValid(schema, out validationEvents))
                 //{
-                    var hBowl = HBowl.FromJson(jsonString);
+                    HBowl hBowl = HBowl.FromJson(jsonString);
```

Comment:
I think using the actual type here makes it easier for quick reference, clear and descriptive

Reply:
Provided HBowl is the correct type, yes... strict typing is good for documentation and readability.